_rules/element-lang-matches-default-language-off6ek.md

```diff
@@ -315,7 +315,7 @@ The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three let
 
 #### Inapplicable Example 7
 
-The `lang` [attribute value][] of this `p` element is a [grandfathered tag][grandfathered tags]. This is not a [valid language tag][].
+The `lang` [attribute value][] of this `p` element is a [grandfathered tag][grandfathered tags]. This is not a [known primary language tag][].
```

Comment:
```suggestion
The `lang` [attribute value][] of this `p` element has no [known primary language tag][] because the `i-lox` [grandfathered tag][grandfathered tags] does not exist in the [language subtag registry][].
```